tests/dbus/network/test_network_manager.py

```diff
@@ -144,16 +144,16 @@ async def test_handling_bad_devices(
 
     with patch.object(DBus, "init_proxy", side_effect=DBusFatalError()):
         await network_manager.update(
-            {"Devices": ["/org/freedesktop/NetworkManager/Devices/100"]}
+            {"Devices": [device := "/org/freedesktop/NetworkManager/Devices/100"]}
```

Comment:
I think this test update doesn't belong here.

Reply:
I had to change the test because previously it was asserting that the log was completely empty. I forget exactly what I changed as part of this that caused the log to not be completely empty but that was an overtest anyway. The test just wanted to confirm that we weren't generating log noise for particular types of devices so it was better to check that the particular log line I wanted was missing.